cmd/bgpfinder-server/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/alistairking/bgpfinder/bgpfinder"
+	"github.com/gorilla/mux"
+)
+
+func main() {
```

Comment:
I think we'll want to have some command line args to control listen port and probably other things. 

Let's also create and use a context so that we can shut down the http server in a controlled fashion (maybe not as important in the super short term, but as we add more things it'll be necessary)
something like

```go
	ctx, stop := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGKILL, syscall.SIGTERM)
	defer stop()
```

It's a bit of a crappy dance to get the http server to play nicely like this, but something along the lines of (this is mostly written from my head, so it mightn't work quite as-is)

```go
  eg := errgroup.WithContext(ctx)
	eg.Go(func() error {
		if err := httpServer.Serve(lnHTTP); err != nil && !errors.Is(err, http.ErrServerClosed) {
			return err
		}
    return nil
	})

	eg.Go(func() error {
		<-ctx.Done()
		shutdownCtx, shutdownCancel := context.WithTimeout(context.Background(), time.Minute)
		defer shutdownCancel()
		if err := httpServer.Shutdown(shutdownCtx); err != nil {
			return err
		}
    return nil
	})
  
  if err := eg.Wait(); err != nil {
    log.Error().Err(err).Msg("HTTP server failed")
  }
```